docs/index.rst

```diff
@@ -136,13 +129,13 @@ If you use PySAL-spaghetti in a scientific publication, we would appreciate usin
     keywords    = {graph-theory,network-analysis,python,spatial-networks,topology}
   }
 
-
 Citing Work
```

Comment:
And have the same list you have above also here?

Reply:
Correct. Maybe only in the doc site is better? Then remove the one in README?

Reply:
Or embed Readme in the documentation like here https://github.com/xarray-contrib/xvec/blob/main/doc/source/index.md?plain=1

Reply:
Although I like that idea, the README and docs/index for `spaghetti` are different enough to make that a pain. I'll remove the *Citing Works* section from README.